server/instructions.md

```diff
@@ -0,0 +1 @@
+Container is available at ghcr.io/securims/ml-prjctr
```

Comment:
would be nice to have a README with full instructions, how to build, how to push, etc

Reply:
done